.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: go
 
 go:
+  - 1.15
```

Comment:
What is this file responsible for? 🤔

Reply:
It's the config file for travis CI (see the checks for this PR, they passed).